### DISCUSSION TITLE
Display chapter progress instead of book progress

### DISCUSSION THREAD

**Discussion by Apollo5555** [advplyr/audiobookshelf-app#822] [Ideas]
I would like the option to be able to switch to progress on the chapter, when audiobook is playing, instead of the progress of the whole book. That why I can have a visible representation of when the chapter will be ending and how much time is left in it.
I looked and couldn't find this feature so making a request.
Using android.
I just started using Audiobookshelf and I love it. Thank you for all the great work that has gone into it.

![Screenshot_20230731_133937_audiobookshelf](https://github.com/advplyr/audiobookshelf-app/assets/115109602/96578947-7676-44c8-8a21-13fd1e781a1d)

**Comment by advplyr** [comment]
Hey if your audiobook has chapters this is supported by opening up the audio player and use the context menu in the top right.

**Reply by Apollo5555** [reply]
I guess I suck at looking. Thank you very much.
